iotdb/src/main/java/org/apache/iotdb/db/engine/storagegroup/TsFileProcessor.java

```diff
@@ -399,6 +404,16 @@ void flushOneMemTable() {
     if (shouldClose && flushingMemTables.isEmpty()) {
       try {
         writer.mark();
+        try {
+          double compressionRatio = totalMemTableSize / writer.getPos();
+          if (totalMemTableSize < writer.getPos()) {
+            logger.error("Compression ratio {} has been less than 1.0.", compressionRatio);
```

Comment:
I think this may happen... not an error status

Reply:
I see, it may be happen. fixed.